### PR TITLE
Add setting user email during creation and update

### PR DESCRIPTION
**Pull request by outcatcher**
## Summary of the Pull Request
Implement setting email with [extended API](https://docs.otc.t-systems.com/en-us/api/iam/iam_08_0011.html)

## PR Checklist

* [x] Refers to: #551
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
=== RUN   TestAccIdentityV3User_basic
--- PASS: TestAccIdentityV3User_basic (29.62s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
Waiting for SDK release

**Comment by outcatcher** [comment]
recheck

File: opentelekomcloud/resource_opentelekomcloud_identity_user_v3.go [hunk 1]
```diff
@@ -109,13 +116,49 @@ func resourceIdentityUserV3Read(d *schema.ResourceData, meta interface{}) error
...
+	identityClient, err := config.identityV3Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("Error creating OpenStack identity client: %s", err)
+	}
+
+	var hasChange bool
+	var updateOpts users.ExtendedUpdateOpts
+
+	if d.HasChange("email") {
+		hasChange = true
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_identity_user_v3.go:150]: you can remove this bool and from 
```go
	if hasChange {
		_, err := users.ExtendedUpdate(identityClient, d.Id(), updateOpts).Extract()
		if err != nil {
			return fmt.Errorf("Error updating OpenStack user: %s", err)
		}
	}
```
to 
```go
	_, err := users.ExtendedUpdate(identityClient, d.Id(), updateOpts).Extract()
	if err != nil {
		return fmt.Errorf("Error updating OpenStack user: %s", err)
	}
```
will be same result

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**